pandas/core/arrays/floating.py

```diff
@@ -258,13 +258,13 @@ def __init__(self, values: np.ndarray, mask: np.ndarray, copy: bool = False):
         super().__init__(values, mask, copy=copy)
 
     def __neg__(self):
-        return type(self)(-self._data, self._mask)
+        return type(self)(-self._data, self._mask.copy())
 
     def __pos__(self):
-        return self
+        return self.copy()
```

Comment:
This is inconsistent with IntegerArray, although it's not clear to me what the correct return type should be. I think we return a copy for a Series, and a new object with the same backing data for TimedeltaArray. numpy returns a new object. python returns the same object with scalars.

maybe just match IntegerArray for now so that we can move these methods to the base class